src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -305,11 +307,12 @@ public interface JavaLangAccess {
     int countNonZeroAscii(String s);
 
     /**
-     * Constructs a new {@code String} by decoding the specified subarray of
-     * bytes using the specified {@linkplain java.nio.charset.Charset charset}.
-     *
-     * The caller of this method shall relinquish and transfer the ownership of
-     * the byte array to the callee since the later will not make a copy.
+     * Constructs a new {@code String} by decoding the specified byte array
+     * using the specified {@linkplain java.nio.charset.Charset charset}.
+     * <p>
+     * <b>WARNING: The caller of this method shall relinquish and transfer the
+     * ownership of the byte array to the callee</b>, since the later will not
```

Comment:
Typo:  "later" -> "latter". (pre-existing)

Reply:
Fixed in f0623258e0506c68a274a1e094039bd6d76e3fa6.